src/components/generator/Sidebar.tsx

```diff
@@ -1,7 +1,10 @@
+import { useState } from 'react'
 import Link from 'next/link'
 import { useRouter } from 'next/router'
 import styled from 'styled-components'
 import { MdDragIndicator } from 'react-icons/md'
+import arrayMove from 'array-move'
+import SortableList, { SortableItem, SortableKnob } from 'react-easy-sort'
```

Comment:
To avoid the following warning, shall we load SortableList dynamically with ssr disabled?

```
Warning: useLayoutEffect does nothing on the server, because its effect cannot be encoded into the server renderer's output format. This will lead to a mismatch between the initial, non-hydrated UI and the intended UI. To avoid this, useLayoutEffect should only be used in components that render exclusively on the client. See https://reactjs.org/link/uselayouteffect-ssr for common fixes.
    at SortableList (/home/ducaale/resumake.io/node_modules/react-easy-sort/index.js:375:21)
```

Something like this should do the trick
```js
const SortableList = dynamic(
  async () => await import('react-easy-sort'),
  { ssr: false }
)
```

Reply:
Not sure, I havent got time to checkout nextjs related things yet. So its your call.